convert2rhel/data/7/x86_64/configs/centos-7-x86_64.cfg

```diff
@@ -9,20 +9,23 @@ gpg_fingerprints = 24c6a8a7f4a80eb5
 # Delimited by any whitespace(s).
 excluded_pkgs =
   anaconda-dracut
-  centos-bookmarks
-  centos-indexhtml
-  centos-logos
   cockpit-subscriptions
   geoipupdate
   kmod-kvdo
-  libreport-centos
-  libreport-plugin-mantisbt
   mod_ldap
   mod_proxy_html
   rhn*
   yum-rhn-plugin
   gnome-documents-libs
 
+# Mapping of packages that need to be swapped during the transaction
+# TODO libreport might be problematic, is there just for testing
```

Comment:
Is it problematic still?

Reply:
I tested that, and it seems it's not any more problematic any more. The libreport-centos has dependency libreport-plugin-mantisbt (not present in rhel repos) but when the mantisbt is removed from the excluded list in config file, the mantisbt removed during the dependency resolution as problematic - so there is no need to have libreport pkgs on excluded list.

The another good thing is, that the libreport-rhel doesn't require the mantisbt package, so it can be successfully swapped and the conversion is successfull.

The log is there: [http://artifacts.osci.redhat.com/testing-farm/1c32b178-5112-4749-85bf-45cf57cf65d0/work-packages_in_system_and_config_centos7_libreport92n17s89/log.txt](http://artifacts.osci.redhat.com/testing-farm/1c32b178-5112-4749-85bf-45cf57cf65d0/work-packages_in_system_and_config_centos7_libreport92n17s89/log.txt) 

I installed libreport-centos before the conversion (also with the mantisbt as it's dependency) and after the conversion the libreport-rhel is present.